build_modules/lib/src/meta_module.dart

```diff
@@ -55,10 +55,10 @@ Module _moduleForComponent(
   return Module(primaryId, sources, directDependencies, platform, isSupported);
 }
 
-Map<AssetId, Module> _entryPointModules(
+Map<AssetId, List<Module>> _entryPointModules(
```

Comment:
This seem weird that the values are lists of modules - at least deserving of a comment if the idea is that they are module groups that will be added to later on? Or maybe rename it?